numpy/core/shape_base.py

```diff
@@ -206,6 +207,7 @@ def vstack(tup):
     dstack : Stack arrays in sequence depth wise (along third dimension).
     concatenate : Join a sequence of arrays along an existing axis.
     vsplit : Split array into a list of multiple sub-arrays vertically.
+    block : Assemble arrays from blocks.
```

Comment:
I think that this phrasing of the description is by far the best one. "Create block arrays" does not really make much sense without going into a lot more detail.